race/Zagorath; Vryloka.json

```diff
@@ -0,0 +1,138 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "Vryloka",
+				"abbreviation": "Vryloka",
+				"full": "Vryloka",
+				"authors": [
+					"u/Zagorath"
+				],
+				"convertedBy": [
+					"p4535992"
+				],
+				"version": "1.1.0",
+				"url": "https://www.reddit.com/r/UnearthedArcana/comments/3tp1bm/vryloka_race_v5_final/",
+				"targetSchema": "1.0.0"
+			}
+		],
+		"dateAdded": 0
+	},
+	"race": [
+		{
+			"name": "Vryloka",
+			"source": "https://www.reddit.com/r/UnearthedArcana/comments/3tp1bm/vryloka_race_v5_final/",
```

Comment:
This should match the `"json"` string you've specified in `"_meta"`, i.e. `"Vryloka"`

Reply:
Gulp. Sorry . First time with this json.
Now it must be correct.